va/va.go

```diff
@@ -45,6 +45,18 @@ const (
 	//   PEBBLE_VA_NOSLEEP=1 pebble
 	noSleepEnvVar = "PEBBLE_VA_NOSLEEP"
 
+	// sleepTimeEnvVar defines the environment variable name used to set the time
+	// the VA should sleep between validation attempts (if not disabled). Set this
+	// e.g. to 5 when you invoke Pebble if you wish the delays to be between 0
+	// and 5 seconds (instead between 0 and 15 seconds):
+	//   PEBBLE_VA_SLEEPTIME=5 pebble
+	sleepTimeEnvVar = "PEBBLE_VA_SLEEPTIME"
+
+	// defaultSleepTime defines the default sleep time (in seconds) between
+	// validation attempts. Can be disabled resp. modified by the environment
```

Comment:
Is "Can be disabled resp. modified by the environment" a typo? Should it read "Can be disabled or modified by the environment"

Reply:
Yes, it should have been *or*. I've moved the *resp* to where the *and* was, I hope it is more correct now :)